receiver/jaegerreceiver/config.go

```diff
@@ -20,9 +20,10 @@ import (
 
 // Config defines configuration for Jaeger receiver.
 type Config struct {
-	TypeVal   string                                      `mapstructure:"-"`
-	NameVal   string                                      `mapstructure:"-"`
-	Protocols map[string]*receiver.SecureReceiverSettings `mapstructure:"protocols"`
+	TypeVal        string                                      `mapstructure:"-"`
+	NameVal        string                                      `mapstructure:"-"`
+	Protocols      map[string]*receiver.SecureReceiverSettings `mapstructure:"protocols"`
+	RemoteSampling map[string]string                           `mapstructure:"remotesampling"`
```

Comment:
Does it make sense to have a more concrete type here instead of a string map?  Something like:
```
type RemoteSamplingConfig struct {
  fetchEndpoint string `mapstructure:fetch-endpoint`
}
```

Reply:
Ah right, thanks!